react/features/prejoin/components/dialogs/CallingDialog.js

```diff
@@ -35,6 +37,30 @@ type Props = {
     t: Function,
 };
 
+const useStyles = makeStyles(theme => {
+    return {
+        root: {
+            padding: theme.spacing(3),
+            textAlign: 'center',
+
+            '& .prejoin-dialog-calling-header': {
+                textAlign: 'right'
+            },
+
+            '& .prejoin-dialog-calling-label': {
+                fontSize: '15px',
+                margin: `${theme.spacing(2)} 0 ${theme.spacing(2)} 0`
```

Comment:
This seems to be different from how it was in SCSS. Is it intentional?

Reply:
No It was a typo. Thank you.